docs-rhino-config.md

```diff
@@ -10,14 +10,11 @@
 
 # Rolling Rhino Remix - rhino-config Documentation
 
-The rhino-config command has been created to extend the capabilities of
-rhino-update. It is not recommended to use rhino-config unless you are aware of
-what options you are enabling. When you run rhino-config you will be prompted
-to choose Yes/No on certain configurations, if you choose "Yes", then the
-changes will be reflected on your system with your next rhino-update. If you
-have already enabled a configuration option, then you will not be prompted
-again for that option. Once an option has been enabled you will not see it
-re-appear in rhino-config.
+`rhino-config` offers two ways of changing your configuration. A
+non-interactive mode, where you can use flags to toggle options in the CLI, and
+an interactive mode which is recommended for newcomers. Interactive mode will
+walk you through the enabling/disabling of features. Once an option has been
+enabled you will not see it re-appear in `rhino-config`.
 
 The rhino-config command line tool created to extend the capabilities of
 rhino-update. It is not recommend to use rhino-config unless you are aware of
```

Comment:
The small description of the rhino-config utility should go before how you can edit your configuration.